setup.py

```diff
@@ -20,10 +21,17 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
+version_namespace = {}
+version_path = convert_path('bleach/version.py')
+with open(version_path) as version_file:
+    exec(version_file.read(), version_namespace)
```

Comment:
I actually had similar thoughts about this. `exec` doesn't feel right, especially in a security package. Will do.